Vagrantfile

```diff
@@ -0,0 +1,118 @@
+  # NOTE: This will enable public access to the opened port
+  config.vm.network "forwarded_port", guest: 5000, host: 80
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine and only allow access
+  # via 127.0.0.1 to disable public access
+  # config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  # config.vm.network "private_network", ip: "192.168.33.10"
+
+  # Create a public network, which generally matched to bridged network.
+  # Bridged networks make the machine appear as another physical device on
+  # your network.
+  # config.vm.network "public_network"
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  # config.vm.synced_folder "../data", "/vagrant_data"
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  # config.vm.provider "virtualbox" do |vb|
+  #   # Display the VirtualBox GUI when booting the machine
+  #   vb.gui = true
+  #
+  #   # Customize the amount of memory on the VM:
+  #   vb.memory = "1024"
+  # end
+  #
+  # View the documentation for the provider you are using for more
+  # information on available options.
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Ansible, Chef, Docker, Puppet and Salt are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  # config.vm.provision "shell", inline: <<-SHELL
+  #   apt-get update
+  #   apt-get install -y apache2
+  # SHELL
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+	sudo apt-get update
+	
+	# Install pyenv prerequisites
+	sudo apt-get install -y build-essential libssl1.0-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+	
+	# Install pyenv
+	git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+	
+	# Set the profile and the env if this is not done before
+	if [ ! $PYENV_ROOT ]
+	then
+		echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+		echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+		echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.profile
+		source ~/.profile
+	fi
+	
+	# Install python 3.9.1
+	pyenv install 3.9.1
+	pyenv global 3.9.1
+	
+	# Install poetry
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+	source ~/.profile
+	
+	# Install gunicorn
+	cd /vagrant
+	poetry run pip install gunicorn
```

Comment:
This means the version of gunicorn isn't source controlled.

I don't think there's any significant cost to just adding gunicorn to the pyproject.toml like any other dependency.

Reply:
Done